src/amp/server/document.test.tsx

```diff
@@ -56,12 +59,17 @@ test('produces valid AMP doc', async () => {
         },
     };
 
+    const activeTests = extractModelAndStyle(
+        getActiveTests(abTestsFullConfig, {"ab-zero-test-experiment": true})
+    )[0];
```

Comment:
why just the first one?

Reply:
The first element of the array is the model, second element is the style. I originally had this as an object but it was adding complexity to the `extractModelAndStyle` function.

Reply:
`extractModelAndStyle` has been refactored into two functions - one for AmpModel, one for Style - to improve clarity